### ISSUE TITLE
Unable to read with Talkback

### ISSUE THREAD

**Issue by zaiff** [barteksc/AndroidPdfViewer#699]
Hi, this library is working great, but its not able to read when Talkback is on. Am I missing something or it is an issue?
